README.md

```diff
@@ -8,31 +8,39 @@ This will prove to be a considerable aid when you need to identify the most impo
 
 ## Week 2
 
 Tinkering Graphics, the Agile Philosophy, and Journal Articles in Computing
 
-Weekly journal goes here. Reflect.
+This week we were divided into pairs to do our tinkering graphics. At first, I wasn't sure whether I could do my best like that. But soon I understood that this is the only way I can learn Python as fast as possible. As I'm new to Python and PyGame in general, I wasn't sure what I was doing, but with the help of my partner, I could at least understand it. In addition, it was so interesting to see how he did different things, experiments, with our simple code. I definitely should spend more time studying individually.
+
+Agile Philosophy was something incomprehensible to me. I can't tell whether I understood it clearly what Agile Philosophy is. Eventually, I assume, I will get it. The only thing to mention is that I don't have to wait until the very last moment, I should research that by myself. I must grow a habit of that...
 
 ## Week 3
 
 Starting my Tinkering Graphics Assignment, Pitching my First Game Concept, and Version Control
 
-Weekly journal goes here. Reflect.
+First thing first, I was so glad to have a chance of presenting my game that I came up with. It wasn't so easy to make gameplay seem interesting, but that what makes it fun to create your own games. The only thing I struggled with a lot was commercials... I had to research how to calculate everything, which wasn't as exciting as everything else. But I'm glad to briefly understand what it looks like. Unfortunately, I didn't give enough effort in presenting my game. That's why it was denied. But that only means I have much to work with. And next time I will surely make a game that will make everyone tremble with delight.
+
+I thought that our Assignment would take quite some time to make. But luckily, I am in a pair with experienced student, that surely knows what he is doing unlike me. Despite that I was trying to do my best even when not knowing some basics of Python. And with my partner's help I can surely tell, that he helped me a lot.
+
+Version Control wasn't as hard as I expected. No doubt I can clearly understand how it works and why it is useful in every development teams.
```

Comment:
Is this a declaration to volunteer to be the go-to person for version control? :1st_place_medal: